packages/cspell-io/package.json

```diff
@@ -33,13 +33,17 @@
   },
   "homepage": "https://github.com/streetsidesoftware/cspell#readme",
   "dependencies": {
+    "@types/node": "^14.14.6",
```

Comment:
There shouldn't be a hard dependency upon the node types.

Reply:
Think I got that from https://github.com/streetsidesoftware/cspell/blob/c8b115db4916f4d31ebfc10972437030e71fac3f/packages/cspell-io/src/file/fileWriter.ts#L6

Reply:
I'll look at cleaning this up.